src/main/java/seedu/ccacommander/model/ModelManager.java

```diff
@@ -190,6 +190,48 @@ public static Enrolment findEnrolmentFromList(List<Enrolment> lastShownEnrolment
         }
     }
 
+    /**
+     *  Edit the Enrolments from the enrolment list which
+     *  contain an event that have the same name as {@param prevName}
+     *  to match the {@param newName} of the event
+     * @param prevName
+     * @param newName
+     * @param model
+     */
+    public static void editEnrolmentsWithEventName(Name prevName, Name newName, Model model) {
+        model.updateFilteredEnrolmentList(PREDICATE_SHOW_ALL_ENROLMENTS);
+        // get the enrolments list then loop through to edit matching enrolments
+        List<Enrolment> enrolmentList = model.getFilteredEnrolmentList();
+        for (Enrolment enrolment: enrolmentList) {
+            if (enrolment.getEventName().equals(prevName)) {
+                Enrolment editedEnrolment = new Enrolment(enrolment.getMemberName(), newName,
+                        enrolment.getHours(), enrolment.getRemark());
+                model.setEnrolment(enrolment, editedEnrolment);
+            }
+        }
+    }
+
```

Comment:
Hmm, any reasoning for this to be a static function? I was thinking that since modelManager implements Model, we can have the modelManager call the function instead of passing in the model as a parameter.

Reply:
okay I will do that :')